.kitchen.docker.yml

```diff
@@ -1,7 +1,7 @@
 driver:
   name: dokken
-  chef_version: latest
   privileged: true # because Docker and SystemD/Upstart
+  chef_version: current
```

Comment:
current is the very latest builds off master. Latest is the latest stable release.